ruby-challenges.rb

```diff
@@ -10,6 +10,21 @@
 reposts3 = 221
 # Expected output: '221 is odd'
 
+def is_even interger
+    if(interger.even? == true)
```

Comment:
Remember the `.even?` returns a Boolean already, so no need to add `== true` to the end